contract_payment_mode/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2015 Domatix (<www.domatix.com>)
+# Copyright 2016 Antiun Ingenieria S.L. - Antonio Espinosa
+# Copyright 2017 Tecnativa - David Vidal
+# Copyright 2017 Tecnativa - Carlos Dauden <carlos.dauden@tecnativa.com>
+# Copyright 2017-2018 Tecnativa - Vicent Cubells <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Contract Payment Mode',
+    'summary': 'Payment mode in contracts and their invoices',
+    'version': '12.0.1.0.0',
+    'author': 'Domatix, '
+              'Tecnativa, '
+              'Odoo Community Association (OCA)',
+    'website': 'http://www.domatix.com',
```

Comment:
@ThomasBinsfeld @aheficent URL should be `https://github.com/OCA/contract`

Reply:
@nikul-serpentcs Done